app/controllers/sign_and_submit_controller.rb

```diff
@@ -5,17 +5,11 @@ def update
     @step = step_class.new(step_params)
 
     if @step.valid?
-      current_app.update!(step_params)
-      send_submission_sms
-      redirect_to(next_path)
+      current_app.update(step_params)
+      flash[:notice] = "Your application has been successfully submitted."
```

Comment:
Probably not in this PR, but we should come up with an I18n plan at some point. (e.g. "I18n every string we touch" or "don't bother until we get to internationalization, when we can do all of it at once")